test-pdfjs.js

```
@@ -0,0 +1,2 @@
+const fs = require('fs');
```

ESLint

[error] 'fs' is assigned a value but never used.

(no-unused-vars)